kuma/wiki/views/list.py

```diff
@@ -12,6 +13,7 @@
 from ..queries import MultiQuerySet
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/all and https://developer.mozilla.org/en-US/docs/tag/* to robots.txt as well?